scripts/README.md

```diff
@@ -0,0 +1,23 @@
+# Scripts
+
+See [https://github.com/github/scripts-to-rule-them-all](https://github.com/github/scripts-to-rule-them-all)
+for additional background on these scripts.
+
+Below is a list of scripts available, along with a simple description of
+what each one does. The details of what they are doing is available within the
+script.
+
+[`pre-bootstrap`](pre-bootstrap)
+Directs towards base development machine setup.
+
+[`bootstrap`](bootstrap)
+Installs project's direct dependencies e.g. npm packages.
+
+[`start`](start)
+Starts the application in a Docker container.
+
+[`test`](test)
+Starts a Docker container specifically for continually running tests.
+
+[`ci-deployment`](ci-deployment)
```

Comment:
This script doesn't exist in the project. However, `test-ci` does - maybe worth adding an entry for that?